### ISSUE TITLE
Add signature help support for .NET methods

### ISSUE THREAD

**Issue by kaylaerdmann** [PowerShell/PowerShellEditorServices#13]
C# methods are using the suggestions documentation window to show parameter hints (see below).
![csharpbug](https://cloud.githubusercontent.com/assets/6333152/9230772/09f9419c-40d9-11e5-83d8-862116da82a7.png).

The information contained this suggestion detail needs to be used in parameter hints. Currently the parameter hints UI does not display for c# methods.

**Comment by daviwil** [comment]
@kapilmb has been working on this already, maybe we can get it in for 0.9.0 in January?

**Comment by kapilmb** [comment]
Yes, I think we will have it merged by then.

**Comment by tylerl0706** [comment]
<img width="1184" alt="screen shot 2018-01-16 at 6 33 35 pm" src="https://user-images.githubusercontent.com/2644648/35022632-cb50b59a-faeb-11e7-92ba-e5800ec4fa8a.png">

This looks like it's working now?

**Comment by daviwil** [comment]
Nah, that's just the "description" of the completion.  Signature help is the thing that comes after you type .Add( and you get the parameter list of the method as a popup above as you type.  This is definitely one of the gaps in PSES.
